lib/transforms/base-js.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 const handlebarsConfig = {
 	debug: false, // set to true to debug finding partial/helper issues
 	extensions: ['.html'],
-	helperDirs: [path.resolve('./node_modules/@financial-times/n-handlebars/src/helpers'), path.resolve('./server/helpers')],
+	helperDirs: [path.resolve('./node_modules/@financial-times/n-handlebars/src/helpers'), path.resolve('./server/helpers'), path.resolve('./node_modules/@financial-times/n-image/src/presenters')],
```

Comment:
will this have problems with things that aren't handlebars helpers but exist in these folders or will it ignore them? we can move files to helpers if it is a problem